src/functions/manifold_functions.jl

```diff
@@ -11,23 +11,62 @@ function ``f: \mathcal M → \mathcal M``, i.e.,
 
 Compute the result in `q`.
 
-see also [`reflect`](@ref reflect(M::AbstractManifold, p, x))`(M,p,x)`.
+see also [`reflect`](@ref reflect(M::AbstractManifold, p, x))`(M,p,x)`, to which the keywords are also passed to.
 """
-reflect(M::AbstractManifold, pr::Function, x) = reflect(M, pr(x), x)
-reflect!(M::AbstractManifold, q, pr::Function, x) = reflect!(M, q, pr(x), x)
+reflect(M::AbstractManifold, pr::Function, x; kwargs...) = reflect(M, pr(x), x; kwargs...)
+function reflect!(M::AbstractManifold, q, pr::Function, x; kwargs...)
+    return reflect!(M, q, pr(x), x; kwargs...)
+end
 
 @doc raw"""
-    reflect(M, p, x)
-    reflect!(M, q, p, x)
+    reflect(M, p, x, kwargs...)
+    reflect!(M, q, p, x, kwargs...)
 
 Reflect the point `x` from the manifold `M` at point `p`, i.e.
 
 ````math
-    \operatorname{refl}_p(x) = \exp_p(-\log_p x).
+    \operatorname{refl}_p(x) = \operatorname{retr}_p(-\operatorname{iretr}_p x).
 ````
 
-where exp and log denote the exponential and logarithmic map on `M`.
+where $\operatorname{retr}$ and $\operatorname{iretr}$ denote a retraction and an inverse
+retraction, respecivelu.
```

Comment:
We use `\operatorname{retr}^{-1}_pq` for inverse retraction.
```suggestion
where $\operatorname{retr}$ and $\operatorname{iretr}$ denote a retraction and an inverse
retraction, respectively.
```